client/GameComponents/Card.jsx

```diff
@@ -436,7 +472,7 @@ class Card extends React.Component {
     render() {
         if(this.props.wrapped) {
             return (
-                <div className={ 'card-wrapper ' + this.getWrapper() } style={ this.props.style }>
+                <div className={ 'card-wrapper' } style={ Object.assign(this.props.style ? this.props.style : {},this.getWrapperStyle()) }>
```

Comment:
Doesn't this mutate `this.props.style`? I thought that was a no no.... I guess you could fix it by adding `{}` as the first arg in the `assign` function?

Reply:
Good catch! props are supposed to be immutable. Will fix.